charts/agent/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.observe.token.create -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: agent-credentials
+type: Opaque
+data:
+  OBSERVE_TOKEN: {{ .Values.observe.token.value | b64enc | quote }}
+  ENTITY_TOKEN: {{ .Values.observe.entityToken.value | b64enc | quote }}
```

Comment:
what is an entity token? and they can provide it via this secret or via the values.yaml?

Reply:
That is the token used to send data to the direct write entity table :)
(Eventually, we should have 3 tokens: one for turnkey logs, one for turnkey metrics, one for direct-write entity.

Reply:
why do we need a separate token? can't we just create the base datastream to be entity table enabled?

Reply:
for posterity: we want it because: https://observeinc.slack.com/archives/C06RY0RQ7TP/p1724788582005679?thread_ts=1724784451.989259&cid=C06RY0RQ7TP